modules/@apostrophecms/image/index.js

```diff
@@ -78,6 +78,24 @@ module.exports = {
       }
     }
   },
+  extendRestApiRoutes: (self, options) => ({
+    async getAll (_super, req) {
+      const pieces = await _super(req);
+
+      console.info(pieces.results.map(piece => ({
+        title: piece.title,
+        _id: piece._id
+      })));
+
+      for (const piece of pieces.results) {
+        piece._path = await self.apos.attachment.url(piece.attachment, {
```

Comment:
Yes we need this, but see `apostrophe-headless` for the way to do it to match our current practice for including it in RESTful responses. We use the `annotate: true` feature of `apos.attachment.all`.

```
            // Attach `_url` and `_urls` properties
            self.apos.attachment.all(pieces, { annotate: true });
```